template_content/README.md.jinja

```diff
@@ -89,6 +89,9 @@ For pull requests and pushes to `main` branch against this repository the follow
 {%- endif %}
 {%- if use_python_pytest %}
  - Python tests are executed using [pytest](https://docs.pytest.org/)
+{%- endif %}
+{%- if use_pre_commit %}
```

Comment:
Just realized, this isn't relevant as part of the Continuous Integration section. Can remove this and move the `pre-commit install` instructions to the next section